gatsby-config.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
     title: 'PatternFly 4 - HTML',
     siteUrl: 'https://pf4.patternfly.org/'
   },
-  pathPrefix: '/patternfly-next',
```

Comment:
just checking - this is no longer needed?

Reply:
Correct! No longer needed.